### PR TITLE
Add nbclassic

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jtpio** [comment]
@conda-forge-admin, please re-render

File: recipes/nbclassic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  description: |
+    Jupyter Notebook as a Jupyter Server Extension.
+  dev_url: https://github.com/jupyterlab/nbclassic
+
+extra:
+  recipe-maintainers:
+    - Zsailer
+    - jasongrout
+    - blink1073
+    - afshin
```
Comment (jtpio) [inline recipes/nbclassic/meta.yaml:48]: cc @Zailer @jasongrout @blink1073 @afshin

Let me know if it's fine to add you as maintainers.

Comment (jasongrout) [reply recipes/nbclassic/meta.yaml:48]: Sure, thanks!

Comment (afshin) [reply recipes/nbclassic/meta.yaml]: Cheers, Jeremy!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**